src/packages/health-check/health-check.context.ts

```diff
@@ -43,6 +44,10 @@ export class UmbHealthCheckContext {
 			this._results.next(undefined);
 		}
 	}
+
+	public destroy(): void {
+		this.destroy();
+	}
```

Comment:
Just leave this function empty - currently it invokes itself :)

```suggestion
	public destroy(): void {
	}
```